recipes/glomap/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "glomap" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/colmap/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: d66b5d36ebc7f0926fe053532598d1d21335f6c3cad34d809c178367489e36ef
+    patches:
+      - colmap-310-compat.patch
+
+build:
+  number: 0
+  ignore_run_exports_from:
+    - libblas
+    - libcblas
+    - colmap
+    - gflags   # [not linux]
+    - poselib  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - cmake
+    - make                           # [unix]
+    - pkg-config
+    - ninja
+    - llvm-openmp                    # [osx]
+    - libgomp                        # [linux]
```

Comment:
Please also add these two to host, too.

Reply:
Thanks for the review @xhochy!

Can I ask if this has always been the case? In the [knowledge base](https://conda-forge.org/docs/maintainer/knowledge_base/#openmp), it only mentions adding them to `build` and there are several recipes in which this is the case; e.g. in colmap which is now leading to version conflicts.

Reply:
The issue is described at https://github.com/conda-forge/openmp-feedstock/issues/126 If it leads to issues here, it means that in one of your dependencies, you also should put it into host.